src/main/java/core/basesyntax/service/strategy/strategyimpl/PurchaseOperationHandler.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.strategy.strategyimpl;
+
+import core.basesyntax.dao.Storage;
+import core.basesyntax.model.FruitRecord;
+
+public class PurchaseOperationHandler implements OperationHandler {
+    @Override
+    public void apply(FruitRecord transaction) {
+        Integer currentQuantity = Storage.storage.get(transaction.getFruit());
```

Comment:
Why in some handlers you check if transaction could be null and in others no? (SupplyOperationHandler)